packages/node/package.json

```
@@ -39,21 +43,18 @@
     "nock": "^13.1.0",
     "prettier": "^2.3.1",
     "rimraf": "^3.0.2",
-    "supertest": "^6.0.1"
+    "supertest": "^6.0.1",
+    "ts-jest": "^27.0.5",
+    "typescript": "^4.4.2"
   },
   "scripts": {
+    "build": "tsc",
     "lint": "eslint .",
-    "pretest": "npm run lint",
-    "prettier": "prettier --list-different --write \"./**/**.js\"",
+    "prepublish": "npm run build",
```

```suggestion
    "prepack": "npm run build",
```

`prepublish` has been deprecated and `prepack` should be the new thing -- I use `prepack` in `oas`.

https://github.com/readmeio/oas/blob/main/package.json#L39
https://docs.npmjs.com/cli/v7/using-npm/scripts#prepare-and-prepublish